### PR TITLE
Package status

### PR DESCRIPTION
**Pull request by daniel-wtd**
# Subject of max. 50 characters

- Update: README.md to reflect new variables
- Update: defaults/main.yml to introduce new variable
- Update: tasks/main.yml to use new wtd_docker_state variable

- Update: docs/*
- Update: tests/*
- Update: .travis.yml
- Update: update-meta-files.sh
- Add: tests/test-whitespace.sh

- Resolves: #11

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 - name: Install docker
```
Comment (Cinux90) [inline tasks/main.yml]: here should be an hint if its got installed (present) or removed (absent)
e.g.: [main.yml of rsnapshot](https://github.com/while-true-do/ansible-role-rsnapshot/blob/master/tasks/main.yml#L3)

**Comment by daniel-wtd** [comment]
Also changed the Service Enable usage.

**Comment by Cinux90** [comment]
Currently it's possible to remove docker and create a group called docker. 
I have no idea for any kind of use case where which can be useful.
I would expect that also the group got remove, if docker got removed.
Thats my opinion. Maybe you have a other one.

Also its not possible to remove the docker group.

**Comment by daniel-wtd** [comment]
Hmmm, you're right.
Normally I would avoid removing configurations, but a group should be ok.

**Comment by daniel-wtd** [comment]
Ah, no there was a reason. The group can be disabled. Since you have to put the packages absent anyways, you should also decide if the group is needed any longer. Maybe there is data written by this group or so.

**Comment by daniel-wtd** [comment]
@Cinux90 anything else?

**Comment by Cinux90** [comment]
Nothing else so far. ;) Approved :+1:
